Cargo.toml

```diff
@@ -29,6 +29,7 @@ raw-window-handle = "0.3"
 [dev-dependencies]
 image = "0.21"
 env_logger = "0.5"
+console_log = "0.1"
```

Comment:
I think it might be good to do to `target.'cfg(target_arch = "wasm32")'.dev-dependencies]` and maybe even with the `sys-web` feature flag but I don't feel too strongly about it.

Reply:
IIRC, it's not possible to check enabled features in the Cargo manifest.

In any case, I have changed to only enable it on Wasm.